packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -213,6 +435,24 @@ it('should take class and style', () => {
   expect(slider.style.width).toBe('350px');
 });
 
+it('should take class and style (vertical)', () => {
```

Comment:
Not needed as it is not vertical orientation specific and is already tested.

Reply:
I can remove it, but just to confirm, would it be safer to test everything we tested for the horizontal cases in the vertical also? Would removing some of these tests from the vertical side make it less robust?

Reply:
**[DONE]** Thanks, was just confirming. Removed this test.

Reply:
@bentleyvk I had one quick question. I ended up just keeping five vertical tests near the end. I chose those because they involved clock coordiates which were vertical specific. All the other vertical tests didn't seem as vertical specific and seemed testable by the horizontal slider case. Can you please confirm if just having the five vertical test cases at the end of the file are okay? Or do I need more test cases?